csv2wiki/__init__.py

```diff
@@ -498,13 +514,9 @@ def __init__(self, level, title=None, column_groups=None):
         represents the header string for column N, but it is the
         caller's responsibility to perform that substitution.
 
-        COLUMN_GROUPS is a list of lists, with each inner list being
-        a column group, and each column in that group corresponding
-        to a cell in this row.  Altogether those columns make up the
-        content of the section, with all the columns in a group being
-        put together on a line, separated by a space between each cell
-        value within that group, and then separated from the next
-        group by a newline."""
+        CONTENT_SPECIFIERS a list of strings, where each represents
+        a section of content that can replace occurrences of "{N}"
```

Comment:
This isn't quite correct.  It is not the case that each string represents one "section" of content; rather, all the strings in `content_specifiers` will be concatenated to form the contents of the single section we are now constructing at `level`.

Reply:
I overloaded the word "section" here I think forgetting that it had a formal meaning above.  I'll use the word "portion" instead.